internal/host/pool_host_statemachine.go

```diff
@@ -27,6 +27,7 @@ func NewPoolHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler)
 			stateswitch.State(models.HostStatusKnownUnbound),
 			stateswitch.State(models.HostStatusUnbinding),
 			stateswitch.State(models.HostStatusUnbindingPendingUserAction),
+			stateswitch.State(models.HostStatusReclaimingPendingReboot),
```

Comment:
PendingReboot sounds like the user needs to do it
```suggestion
			stateswitch.State(models.HostStatusReclaimingRebooting),
```